scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -25,6 +25,30 @@ label {
   fieldset[disabled] & {
     color: $crm-darker-grayblue;
   }
+
+  &[type="text"] {
+    &::-webkit-input-placeholder {
+      /* WebKit, Blink, Edge */
+      color: $crm-dark-grayblue;
```

Comment:
Remove all these comments, it's evident enough what browser each selector is for, it's in the prefix